Index: robo/src/Task/Composer/Base.php
===================================================================
--- robo.orig/src/Task/Composer/Base.php
+++ robo/src/Task/Composer/Base.php
@@ -1,10 +1,11 @@
 <?php
 namespace Robo\Task\Composer;
 
+use Robo\Contract\CommandInterface;
 use Robo\Task\BaseTask;
 use Robo\Exception\TaskException;
 
-abstract class Base extends BaseTask
+abstract class Base extends BaseTask implements CommandInterface
 {
     use \Robo\Common\ExecOneCommand;
 
